utils/fasm/test/test_fasm.cpp

```diff
@@ -260,7 +261,7 @@ TEST_CASE("fasm_integration_test", "[fasm]") {
 
                 // Add additional features to edges that go to CLB.I[11:0] pins
                 // to correlate them with features of CLB input mux later.
-                auto sink_type = device_ctx.rr_nodes[sink_inode].type();
+                auto sink_type = rr_graph.node_type(RRNodeId(sink_inode));
```

Comment:
(In all these places where we now manually convert an integer of sorts into an RRNodeId, we should see that the place we get it from already returns an RRNodeId in the future; but I suppose that is somewhere on the TODO list)

Reply:
Yes, I believe the plan is to do that in the future. With the potential overhead of changing many functions to accept RRNodeIds instead of ints, we are just casting the node_id for now.